src/config/wmodules-ciscat.c

```diff
@@ -172,6 +180,17 @@ int wm_ciscat_read(const OS_XML *xml, xml_node **nodes, wmodule *module)
             ciscat->java_path = strdup(nodes[i]->content);
         } else if (!strcmp(nodes[i]->element, XML_CISCAT_PATH)) {
             ciscat->ciscat_path = strdup(nodes[i]->content);
+        } else if (!strcmp(nodes[i]->element, XML_CISCAT_BINARY)) {
+            ciscat->ciscat_binary = strdup(nodes[i]->content);
```

Comment:
Free the old string (having the default value) before setting a new one.
```suggestion
            os_free(ciscat->ciscat_binary);
            ciscat->ciscat_binary = strdup(nodes[i]->content);
```